packages/showcases/css/stories/components/navbar/examples/overview.html

```diff
@@ -0,0 +1,107 @@
+<div class="block">
+  <nav class="vtmn-navbar">
+    <div class="vtmn-navbar_left-navigation">
+      <div></div>
+    </div>
+
+    <div class="vtmn-navbar_logo">
+      <svg height="40px" viewBox="0 0 200 50" aria-label="Decathlon" role="img">
```

Comment:
```suggestion
      <svg viewBox="0 0 200 50" aria-label="Decathlon" role="img">
```

We should let the height prop in the css file. Not worth it to let it here.

Reply:
Also the image in the navbar is commonly used as a link to the main page. I think we whould consider this usage and make the logo a link.